ScnSoft-Education/LeetCodeTraining.Tests/ContainerWithMostWaterTests.cs

```diff
@@ -0,0 +1,19 @@
+using LeetCodeTraining.ContainerWithMostWater;
+
+namespace LeetCodeTraining.Tests
+{
+    public class ContainerWithMostWaterTests
+    {
```

Comment:
apply "format document" always before commit your code
In Visual studio -> Edit -> Advanced -> format document
or use hot key ctrl + k, d